### PR TITLE
Feat/profile internationalization

### PR DESCRIPTION
**Pull request by aeschi**

### CODE REVIEW

File: src/i18n/content-types.ts [hunk 1]
```diff
@@ -53,6 +53,28 @@ interface Navbar {
 			changePassword: string;
 			deleteAccount: string;
 			approve: string;
+			usernameShould: string;
+			checkInput: string;
+			validateLength: string;
```
Comment (raphael-arce) [inline src/i18n/content-types.ts:58]: I'd suggest to make it consistent with `passwordLength`, let's use `usernameLength` here  (which also makes it easier to understand imo)

File: src/i18n/content-types.ts [hunk 2]
```diff
@@ -53,6 +53,28 @@ interface Navbar {
...
+			register: string;
+			existingAccount: string;
+			logIn: string;
+			passwordShould: string;
+			passwordLength: string;
+			passwordUpperAndLowerCase: string;
+			passwordSpecialChar: string;
+			passwordNumber: string;
+			logInShort: string;
+			missinAccount: string;
```
Comment (raphael-arce) [inline src/i18n/content-types.ts:71]: typo: `missing`

File: src/i18n/content-types.ts [hunk 3]
```diff
@@ -53,6 +53,28 @@ interface Navbar {
 			changePassword: string;
 			deleteAccount: string;
 			approve: string;
+			usernameShould: string;
+			checkInput: string;
+			validateLength: string;
+			onlyNumberAndLetters: string;
+			userOccupied: string;
```
Comment (raphael-arce) [inline src/i18n/content-types.ts:60]: let's try to stick to one way of naming things, in the component this is named `usernameTaken`, i'd suggest to use the same name here

File: src/i18n/i18n-store.ts [hunk 4]
```diff
@@ -19,7 +19,7 @@ interface I18NState {
 }
 
 export const useI18nStore = create<I18NState>()((set, get) => ({
-	language: "de",
+	language: "en",
```
Comment (raphael-arce) [inline src/i18n/i18n-store.ts]: is this on purpose?

**Review by raphael-arce — CHANGES_REQUESTED** [review]
I have a couple of suggestions 👍

File: src/i18n/i18n-store.ts (hunk 4 above)
Comment (aeschi) [reply src/i18n/i18n-store.ts]: I just changed it for testing. Changed it back now

**Review by raphael-arce — APPROVED** [review]
lgtm 🚀

**Pull request merged**